iml-system-test-utils/src/docker.rs

```diff
@@ -0,0 +1,126 @@
+use crate::{iml, CheckedStatus};
+use iml_systemd;
+use std::{io, thread, time};
+use tokio::{
+    fs::{canonicalize, File},
+    io::AsyncWriteExt,
+    process::Command,
+};
+
+const IML_DOCKER_PATH: &str = "/etc/iml-docker";
+
+pub async fn docker() -> Result<Command, io::Error> {
+    let mut x = Command::new("docker");
+
+    let path = canonicalize(IML_DOCKER_PATH).await?;
+
+    x.current_dir(path);
+
+    Ok(x)
+}
+
+pub async fn deploy_iml_stack() -> Result<(), io::Error> {
+    iml_systemd::start_unit_with_time("iml-docker.service".into(), 400).await?;
+
+    Ok(())
+}
+
+pub async fn remove_iml_stack() -> Result<Command, io::Error> {
+    iml_systemd::stop_unit("iml-docker.service".into()).await?;
+
+    let mut x = docker().await?;
+
+    x.arg("stack").arg("rm").arg("iml");
+
+    Ok(x)
+}
+
+pub async fn system_prune() -> Result<(), io::Error> {
+    let mut x = docker().await?;
+
+    x.arg("system")
+        .arg("prune")
+        .arg("--force")
+        .arg("--all")
+        .checked_status()
+        .await?;
+
+    Ok(())
+}
+
+pub async fn volume_prune() -> Result<(), io::Error> {
+    let mut x = docker().await?;
+
+    x.arg("volume")
+        .arg("prune")
+        .arg("--force")
+        .checked_status()
```

Comment:
Will it prune all docker volumes, even the ones unrelated to IML?

Reply:
Yes